src/System.Buffers/src/System/Buffers/ArrayPoolEventSource.cs

```diff
@@ -22,7 +21,7 @@ internal enum BufferAllocationReason : int
             PoolExhausted
         }
```

Comment:
@vancem, seems like the Guid should be removed?  And I assume we should make the Name match the name of the type?  Anything else we should clean up here while I'm at it?